Frontend of Merchant Verification

vahinim opened the pull request:
Add Verification application that verifies Merchant with 2 basic questions.

vahinim committed:
Frontend of Merchant Verification

vahinim committed:
Changed success message

trblessy commented on frontend/verification/lib/VerificationPage2.dart:
```diff
@@ -0,0 +1,249 @@
+import 'package:verification/verificationSuccess.dart';
+import 'package:verification/verificationFailed.dart';
+
+//class verificationPage2 extends StatelessWidget {
```
Please remove commented code.

vahinim replied:
removed

trblessy commented on frontend/verification/lib/VerificationPage2.dart:
```diff
@@ -0,0 +1,249 @@
+//  }
+//}
+
+class VerificationPage2 extends StatefulWidget {
```
Add class level comment as to what it consists and does.
https://dart.dev/guides/language/effective-dart/documentation#consider-writing-a-library-level-doc-comment

vahinim replied:
done

trblessy commented on frontend/verification/lib/VerificationPage2.dart:
```diff
@@ -0,0 +1,249 @@
+import 'package:flutter/material.dart';
```
Name this directory according to

https://dart.dev/guides/language/effective-dart/style#do-name-libraries-and-source-files-using-lowercase_with_underscores

Give more descriptive names, like business_registration_details_view, business_address_view, etc. to this file and others

vahinim replied:
changed .dart file names

trblessy commented on frontend/verification/lib/VerificationPage2.dart:
```diff
@@ -0,0 +1,249 @@
+//  }
+//}
+
+class VerificationPage2 extends StatefulWidget {
```
Please give more descriptive name eg. VerificationHomeView, VerificaitonSuccessView.

vahinim replied:
done

vahinim replied:
done

trblessy commented on frontend/verification/lib/VerificationPage2.dart:
```diff
@@ -0,0 +1,249 @@
+      //When ever user clicks on discard verification it opens up verification failed
+      //TODO: Need to return to home page after cancelling verification
+      _showCancelVerificationDialog();
+
```
remove extra newline.
try formatting using
https://dart.dev/guides/language/effective-dart/style#do-format-your-code-using-dartfmt

vahinim replied:
done

trblessy commented on frontend/verification/lib/VerificationPage2.dart:
```diff
@@ -0,0 +1,249 @@
+    );
+  }
+
+  void _verifyMerchant(){
```
https://dart.dev/guides/language/effective-dart/documentation#consider-writing-doc-comments-for-private-apis

vahinim replied:
done

trblessy commented on frontend/verification/lib/VerificationPage2.dart:
```diff
@@ -0,0 +1,249 @@
+    }
+  }
+
+  Future<void> _showVerifyDialog() async {
```
https://dart.dev/guides/language/effective-dart/documentation#consider-writing-doc-comments-for-private-apis

vahinim replied:
done

trblessy reviewed (commented):
By convention, the description of the PR is a complete sentence, written as though it were an order
'Add dart files for registration views of the Gpay merchant app'

followed by more description of the exact changes

vahinim committed:
Remove commented lines in VerificationPage2.dart

Remove commented code that is not used.
Remove unrelevant comments.

vahinim committed:
Change file names of .dart files.

vahinim committed:
change class names and remove commented code in bussiness_registration_details_view.dart

vahinim committed:
change class names in .dart files

change class name in bussiness_verification_menu_items.dart
change class name in bussiness_verification_view.dart
removed @override in VerificationHomeView in bussiness_verification_view.dart

vahinim committed:
change class names and organize code in .dart files

change class names in business_verification_success_view.dart
change class names in business_verification_failure_view.dart
remove MaterialApp in business_verification_success_view.dart
remove MaterialApp in bussiness_verification_failure_view.dart
remove unused imports in .dart files
shift VerificationStartView class to main.dart
remove menu in business_registration_details_view.dart

vahinim committed:
Add comments to .dart files

Add comments to business_registration_details_view.dart
Add comments to business_verification_view.dart
Add comments to business_verification_success_view.dart
Add comments to business_verification_failure_view.dart
Add comments to business_verification_menu_items.dart
Add comments to main.dart

vahinim committed:
Reformat .dart files

vahinim commented:
Please review again. I have completed the above mentioned changes.

vahinim committed:
Merge branch 'master' into frontend_agent_verification

vahinim committed:
Remove Merge conflicts in Agent application

Move verification/lib/ directory files to gpay_agent/lib/business_verification_views
Change views to agent_views in gpay_agent/lib/
Change datamodels to agent_datamodels in gpay_agent/lib
Add assets directory in gpay_agent
Move gpay_agent/images/ to gpay_agent/assets/agent_beginning_images
Move verification/assets/images to gpay_agent/assets/verification_images
Add google-fonts dependency in gpay_agent/pubspec.yaml
Add font theme to Montserrat in gpay_agent/main.dart
Remove verification/main.dart
Remove all other files in verification
Refactor image path where ever image is used
Modify assets in gpay_agent/pubspec.yaml to present directories
Refactor .dart file imports based on new naming
Add MerchantHomeView in business_registration_views.dart when ever agent clicks on verify in merchant_found_notfound.dart

vahinim commented:
I have removed Merge conflicts. Please review again and then I will merge.

vahinim committed:
Merge branch 'master' into frontend_agent_verification

# Conflicts:
#	frontend/gpay_agent/lib/main.dart
